config/initializers/devise.rb

```diff
@@ -0,0 +1,311 @@
+# frozen_string_literal: true
```

Comment:
suggestion: Let's remove the comments for all configuration options that we do not use.

Reply:
ok

Reply:
@PriyangaPKini Did you end up doing this? I don't see the changes in the diff, so wanted to confirm.

Reply:
I removed some of them.  I find the comments helpful sometimes because I can easily search for the default configuration and can customize it as needed. So I thought of keeping it. But If that is not a good practice I will remove them.

Reply:
hmm, it's always better to explicitly figure out the configuration required (as a learning exercise), but it's not a big deal.